scripts/docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:16.04
+ARG no_proxy
+ARG http_proxy
+ARG https_proxy
+ENV no_proxy ${no_proxy}
+ENV http_proxy ${http_proxy}
+ENV https_proxy ${https_proxy}
+
+ARG CMAKE_URL
+ENV CMAKE_URL ${CMAKE_URL:-https://cmake.org/files/v3.8/cmake-3.8.2-Linux-x86_64.sh}
```

Comment:
in CI we are using cmake 3.9, we need to think about how we can keep this in sync with our CI images hosted at https://github.com/zephyrproject-rtos/ci-dockerfiles